query-node/schema.graphql

```diff
@@ -204,9 +201,11 @@ type Channel @entity {
   "Runtime entity identifier (EntityId)"
   id: ID!
 
-  # "Owner of the channel" Commenting out this field: 'owner' can be curator_group, lead
-  # or a member. We are not handling events related to curator group so we will not set this field
-  # owner: Member!
+  "Member owning the channel (if any)"
```

Comment:
Really, the representation here should be a variant, rather than two separate fields for members and curator group, because you are either owned by a member or by a curator group (or DAO as well actually, bu they do not exist yet). However, if we use that representation, you cannot do reverse lookups into the variant, as you are doing now. This is a missing feature:

https://github.com/Joystream/hydra/issues/187.

So we are probably better off just keeping this for now, but be aware of this.

Reply:
I'm aware of that. This approach was selected to overcome the missing feature you mentioned.